### PR TITLE
Documentation using paradox plugin

### PR DESCRIPTION
**Pull request by sleipnir**

### CODE REVIEW

File: docs/src/main/paradox/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Dart
```
Comment (pvlugter) [inline docs/src/main/paradox/index.md]: Let's have the main title as `Cloudstate Dart`, which is the pattern we're using for other modules like Go and Python.

**Review by pvlugter — COMMENTED** [review]
Cool to see these docs moved already 👍 

Note that there are no git tags currently in the repo, so the doc version is `0.0.0`.

Here's the encrypted deploy token for this repo:

```
# Docs deploy token encrypted with: travis encrypt --pro -r cloudstateio/dart-support DEPLOY_DOCS_TOKEN={token}
secure: "PRMJYFNQBMV8YxvnnKfmF6WbnerRnQzSUNCBVkZYDn69272Iu1Xgmk8ql1z7rek+5kYLy+fEJR+1Go/o6mp0F21w1xq4myjnqiEfd+z9W76RHIFFl/eV9Ud0DZfpxezeukk8FYvkciwVHIykzS/lZr/tdr0tpdb0UAP6LmTZVm/BDYsuuHayZzXcpqNiKyd+lm9e0lhKhC/26clvU7KF4sEh3ApMxd0NwefJL/JvQKxPGOOO83yssxn59iF5oGBp97jGXctPrV9/ShdJETZl4xBVqD+moc8b6AFnAai+EP1p+utJEChz8xRJzgbiKkG/9arcBx89lXZtx3iaKLZ4vnomKUX9MwaGg8jfDYvBaZX73n2xXJC4EvsEYFsdTd0etudru5s/IGQe2126JykDqGXVL8D8Fv8tFhOgXQ9oDSBfHz73fp6dZiAkG+VxmxLoKsha2uyRzDNSvuiQzY52eeAMAkEX6aLUhUzaNQE+7PHWGPxHg4tYMtbkmglvpJ3vvnKSBt0+Jo/zTnF9mrxfsQtj1uJK6qTPcPrfsRgnCZVLi0MyIILpyFOL28+DreplVJ5EU0SQhtToTl8IQQMFK++aHO/Nod3jpxyaGFYXxS7cMMUsN5REAijhMWnHfN92UsT04UBc+tGGkicTlfor5vEfhe6OnbsXvEqMeOFENBQ="
```

File: docs/src/main/paradox/index.md (hunk 1 above)
Comment (sleipnir) [reply docs/src/main/paradox/index.md]: Thanks @pvlugter

**Comment by sleipnir** [comment]
@marcellanz  as nothing new has been changed in the documentation, just changed the place I will follow with this PR. Thank you

**Pull request merged**